hyrisecockpit/database_manager/job/ping_hyrise.py

```diff
@@ -0,0 +1,12 @@
+"""This job checks if the hyrise is still alive."""
+from psycopg2 import DatabaseError, InterfaceError
+
+
+def ping_hyrise(connection_factory, hyrise_active) -> None:
+    """Check in interval if hyrise is still alive."""
```

Comment:
```suggestion
    """Check whether Hyrise is alive."""
```
... since the interval is not at the job level.